src/Status.ts

```diff
@@ -1,108 +1,226 @@
+    public static readonly TransactionTooManyLayers = new Status(65);
+    public static readonly ContractDeleted = new Status(66);
+    public static readonly PlatformNotActive = new Status(67);
+    public static readonly KeyPrefixMismatch = new Status(68);
+    public static readonly PlatformTransactionNotCreated = new Status(69);
+    public static readonly InvalidRenewalPeriod = new Status(70);
+    public static readonly InvalidPayerAccountId = new Status(71);
+    public static readonly AccountDeleted = new Status(72);
+    public static readonly FileDeleted = new Status(73);
+    public static readonly AccountRepeatedInAccountAmounts = new Status(74);
+    public static readonly SettingNegativeAccountBalance = new Status(75);
+    public static readonly ObtainerRequired = new Status(76);
+    public static readonly ObtainerSameContractId = new Status(77);
+    public static readonly ObtainerDoesNotExist = new Status(78);
+    public static readonly ModifyingImmutableContract = new Status(79);
+    public static readonly FileSystemException = new Status(80);
+    public static readonly AutorenewDurationNotInRange = new Status(81);
+    public static readonly ErrorDecodingBytestring = new Status(82);
+    public static readonly ContractFileEmpty = new Status(83);
+    public static readonly ContractBytecodeEmpty = new Status(84);
+    public static readonly InvalidInitialBalance = new Status(85);
+    public static readonly InvalidReceiveRecordThreshold = new Status(86);
+    public static readonly InvalidSendRecordThreshold = new Status(87);
+    public static readonly AccountIsNotGenesisAccount = new Status(88);
+    public static readonly PayerAccountUnauthorized = new Status(89);
+    public static readonly InvalidFreezeTransactionBody = new Status(90);
+    public static readonly FreezeTransactionBodyNotFound = new Status(91);
+    public static readonly TransferListSizeLimitExceeded = new Status(92);
+    public static readonly ResultSizeLimitExceeded = new Status(93);
+    public static readonly NotSpecialAccount = new Status(94);
+    public static readonly ContractNegativeGas = new Status(95);
+    public static readonly ContractNegativeValue = new Status(96);
+    public static readonly InvalidFeeFile = new Status(97);
+    public static readonly InvalidExchangeRateFile = new Status(98);
+    public static readonly InsufficientLocalCallGas = new Status(99);
+    public static readonly EntityNotAllowedToDelete = new Status(100);
+    public static readonly AuthorizationFailed = new Status(101);
+    public static readonly FileUploadedProtoInvalid = new Status(102);
+    public static readonly FileUploadedProtoNotSavedToDisk = new Status(103);
+    public static readonly FeeScheduleFilePartUploaded = new Status(104);
+    public static readonly ExchangeRateChangeLimitExceeded = new Status(105);
+
+    public readonly value: number;
+
+    public constructor(value: number) {
```

Comment:
I'd mark this constructor as private.

Reply:
Constructor is used in other file which is why it's public.

Reply:
Should still add some kind of doc there stating it's "unstable" or otherwise not intended for public use.